### DISCUSSION TITLE
Statistics: Add more, easily separable colors to the diagrams

### DISCUSSION THREAD

**Discussion by jeyemwey** [Traewelling/traewelling#2958] [Ideas / Feature Requests]
If you have a statistic with a large number of different slices, the colors are repeated pretty easily:

<img width="353" alt="image" src="https://user-images.githubusercontent.com/2796271/208897063-484686ed-aada-47d0-9c61-44c367d71e87.png">

AC:
* Add more colors to the wheel, so that the repetition doesn't kick in so easily

**Comment by MrKrisKrisu** [comment]
Any suggestions? In my opinion the current colors are good.
But as of Issue #1768 there are not enough for many columns.

Some [tool](https://venngage.com/tools/accessible-color-palette-generator) suggested these containing the Trwl Color (#C72730), but I am not happy with this examples.

![image](https://github.com/Traewelling/traewelling/assets/4103693/fc990c1a-f554-478f-9633-312945bd663f)

**Comment by MrKrisKrisu** [comment]
Oh sorry, my mistake. The issue is not about contrasting colors, but about the number of colors.

**Comment by pascalwittler** [comment]
I often use https://coolors.co to find matching colors. Here some examples of generated palettes with 10 colors:

`#c72730` `#f0a202` `#f18805` `#202c59` `#d0e3cc`
`#f7ffdd` `#878e88` `#ffffff` `#76bed0` `#f3dad8`

`#c72730` `#023c40` `#0ad3ff` `#78ffd6` `#e1faf9`
`#ffe74c` `#6bf178` `#e7e6f7` `#e3d0d8` `#aea3b0`

`#c72730` `#090809` `#3c91e6` `#a2d729` `#fafffd`
`#db995a` `#d6d4a0` `#ccc7b9` `#eaf9d9` `#e2d4ba`

**Comment by pascalwittler** [comment]
Mögliche Überschneidung mit #2813
